cyfronet-fid/sat4envi
# [#231] Rename Product and ProductType

## What is in this PR?

* Rename Product to Scene
* Rename ProductType to Product
* Fix foreign keys default values

The changes are done in separate commits to contain the rename better.

## How to test?

Run the application in development.
The front-end should work as before, ie. load products, scenes, show them etc.
The front- and back-end tests should pass, I tried to rename variables and names where possible, if I missed something give me heads up.
@michal-szostak please have a look if I didn't mess something up in the front-end

The db is migrated to the new names, the names of the tables and columns reflect the changes - check if they are correct.
Moreover, notice that a number of sequences is reduced - foreign keys were incorrectly defined as BIGSERIAL, which resulted in unused sequences.

Closes: #231.
